dags/multipod/configs/mxla_collective_config.py

```diff
@@ -50,7 +50,7 @@ def get_mxla_collective_config(
   current_date = current_time.strftime("%Y-%m-%d")
   current_datetime = current_time.strftime("%Y-%m-%d-%H-%M-%S")
 
-  base_output_directory = f"{gcs_bucket.XLML_OUTPUT_DIR}/multipod/mxla/nightly/automated/{current_date}/{num_slices}slice-V{tpu_version.value}_{tpu_cores}-mxla-collective-{bytes_to_transfer}transferBytes-{current_datetime}"
+  base_output_directory = f"{gcs_bucket.OUTPUT_DIR}/multipod/mxla/nightly/automated/{current_date}/{num_slices}slice-V{tpu_version.value}_{tpu_cores}-mxla-collective-{bytes_to_transfer}transferBytes-{current_datetime}"
```

Comment:
`current_datetime` will keep updating every time the file is been imported. Is the datetime used as an unique id to prevent the file using the same name as other runs?

Reply:
Yeah, @jonb377 do you know if your team meet any issue for this? We'd like to recommend to have a similar change for GCE track. Also, add this similar feature for xpk (happy to work together on this).

Reply:
This is only safe for the specific use case - the output directory isn't reused across tasks, so whatever value is in the DAG at execution time is what the workload uses. No downstream tasks depend on what value the workload used.

We will be moving to the infra-generated buckets soon.

Reply:
SG! Thank you!